### PR TITLE
RenderModelEditor render fix

### PR DESCRIPTION
**Pull request by AbitDeveloper**
I did this change on my local machine because RenderModel component (rendered by RenderModelEditor.cs) was not rendering properly.

It probably threw an exception silently during render (OnInspectorGUI).

### CODE REVIEW

File: unity_package/Assets/SteamVR/Editor/SteamVR_RenderModelEditor.cs [hunk 1]
```diff
@@ -86,7 +86,7 @@ public override void OnInspectorGUI()
 
 		GUILayout.BeginHorizontal();
 		GUILayout.Label("Model Override");
-		var selected = EditorGUILayout.Popup(renderModelIndex, renderModelNames);
+		var selected = EditorGUILayout.Popup(renderModelIndex, renderModelNames ?? new string[0]);
```
Comment (AbitDeveloper) [inline unity_package/Assets/SteamVR/Editor/SteamVR_RenderModelEditor.cs]: It has (probably) problem when 2nd parameter (renderModelNames) being null.

`renderModelNames ?? new string[0]`

is equivalent to 

`renderModelNames == null ? renderModelNames : new string[0]`

**Comment by aleiby** [comment]
Do you have a way to reproduce this?

OnEnable should be called before OnInspectorGUI and will set renderModelNames using LoadRenderModelNames which always returns an array of strings with at least one entry ("None").

If LoadRenderModelNames is throwing an exception, I'd like to narrow that down and catch that instead.  If OpenVR fails to initialize, we should catch that when checking if holder.instance is null.

If SteamVR_RenderModel.RenderModelInterfaceHolder is throwing an exception, I'd like to fix it there instead since that is used in multiple places.

Thanks!

**Comment by AbitDeveloper** [comment]
It is probably because I am on Linux (=without dll ?). It does not throw any exception in this method (simply returns null).

**Comment by aleiby** [comment]
We should have proper Linux support in Unity soon so I'll verify this all works there.

**Pull request closed without merging**